packages/eslint-plugin/package.json

```diff
@@ -39,7 +39,7 @@
 		"eslint-config-prettier": "^7.1.0",
 		"eslint-plugin-import": "^2.23.4",
 		"eslint-plugin-jest": "^24.1.3",
-		"eslint-plugin-jsdoc": "^34.1.0",
+		"eslint-plugin-jsdoc": "^36.0.8",
```

Comment:
As far as I can tell the only braking change that impacts the WordPress plugin is from v35:

https://github.com/gajus/eslint-plugin-jsdoc/compare/v34.8.2...v35.0.0

> `multiline-blocks`: Should have been enabled by default

See https://github.com/gajus/eslint-plugin-jsdoc/commit/b81e25d14d05c5c65992c9e6f34386fdf14e8d8d.

If we set `jsdoc/multiline-blocks` and `jsdoc/no-multi-asterisks` at the warning level, the we can consider this PR as adding new features only. What do you think?

Reply:
I think you meant https://github.com/gajus/eslint-plugin-jsdoc/compare/v34.8.2...v36.0.0, right? Or do you think we should update until the v35 only?

Reply:
v36 dropped only support for Node 10 or something like that which is isn't relevant in our context.